src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -321,6 +321,10 @@ export const CommentContainer: FunctionComponent<Props> = ({
     comment.author.badges.length > 0
       ? comment.author.badges
       : null;
+  const badgesJoin = badges
+    ?.map((b: string) => `badge-${b.replace(" ", "-")}`)
```

Comment:
Because every "official" classname so far has a `coral-` prefix, maybe we want to do that here too?

Reply:
Agreed, it is done!